Analysis/include/AnaTuple.h

```diff
@@ -45,8 +45,17 @@ namespace analysis {
     VAR(bool, is_central_es) \
     VAR(ULong64_t, sample_id) \
     VAR(double, weight) /* weight */ \
+    VAR(double, btag_weight) /* btag_weight */ \
```

Comment:
- use float to store btag weight
- store btag weight for 3 WP and also up/down values => 3*3 = 9 weights